controllers/managedfusion_controller.go

```diff
@@ -231,6 +236,12 @@ func (r *ManagedFusionReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, fmt.Errorf("%s.smtp_config is not in a valid yaml format: %v", managedFusionSecretName, err)
 	}
 
+	if config, found := r.managedFusionSecret.Data["remote_write_config"]; !found {
+		log.Info("Remote write config not present in the managed fusion secret")
+	} else if err := yaml.Unmarshal(config, r.remoteWriteConfig); err != nil {
+		return ctrl.Result{}, fmt.Errorf("%s.remote_write_config is not in a valid yaml format: %v", managedFusionSecretName, err)
+	}
```

Comment:
As this is an optional section, why do we need to log a message in the log on each reconciliation?
```suggestion
	if config, found := r.managedFusionSecret.Data["remote_write_config"]; found {
		r.remoteWriteConfig = &remoteWriteConfig{}
		if err := yaml.Unmarshal(config, r.remoteWriteConfig); err != nil {
			return ctrl.Result{}, fmt.Errorf("%s.remote_write_config is not in a valid yaml format: %v", managedFusionSecretName, err)
		}
	}
```